src/libs/kata-types/src/mount.rs

```diff
@@ -49,6 +55,22 @@ impl Mount {
     }
 }
 
+/// DirectVolumeMountInfo contains the information needed by Kata
+/// to consume a host block device and mount it as a filesystem inside the guest VM.
+#[derive(Debug, Clone, Default, serde::Serialize, serde::Deserialize)]
```

Comment:
nit(for style alignment): We normally use `use serde::{Serialize, Deserialize}`, and `#[derive(Serialize, Deserialize)`